src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -7930,23 +7933,10 @@ private List<ExpressionAst> InvokeParamParenListRule(Token lParen, out IScriptEx
 
                         if (argumentName is not null)
                         {
-                            if (comma is null)
-                            {
-                                comma = NextToken();
-                                UngetToken(comma);
-                            }
-
-                            string nextTokenText = comma.Text;
-                            if (nextTokenText.Length == 1 && char.IsControl(nextTokenText[0]))
-                            {
-                                nextTokenText = string.Format("\\u{0:x4}", (short)nextTokenText[0]);
-                            }
-
-                            ReportIncompleteInput(After(argumentName.Extent),
-                                nameof(ParserStrings.MissingArgumentAfterLabel),
```

Comment:
Does this entry need to be removed in the resources? I don't think it was used anywhere else.

Reply:
You are right, I forgot to look in the resources. I've removed it now.